source/_posts/angular-authentication-with-openid-connect.md

```diff
@@ -0,0 +1,403 @@
+---
+title: Angular Authentication with OpenID Connect (OIDC)
+author: Viktors Garkavijs
+link: https://mixefy.jp
```

Comment:
Hey @gaeshi - this link seems to be broken, isn't it?

Reply:
The link is correct, I just don't have anything on the website yet.